cognite/powerops/custom_modules/power_model_v1/data_models/views/obligations/DayAheadProductionObligationForBenchmarking.view.yaml

```diff
@@ -0,0 +1,45 @@
+space: "{{power_ops_models}}"
+externalId: DayAheadProductionObligationForBenchmarking
+name: DayAheadProductionObligationForBenchmarking
+description: Production obligations for the day ahead market used in benchmarking
+filter:
+  and:
+    - hasData:
+        - type: container
+          space: "{{power_ops_models}}"
+          externalId: DayAheadProductionObligationForBenchmarking
+    - equals:
+        property:
+          - node
+          - space
+        value: "{{power_ops_instance_space}}"
+    - equals:
+        property:
+          - node
+          - type
+        value:
+          space: "{{power_ops_type_space}}"
+          externalId: DayAheadProductionObligationForBenchmarking
+implements:
+  - space: "{{power_ops_models}}"
+    externalId: BidSource
+    version: "{{version}}"
+    type: view
+version: "{{version}}"
+properties:
+  timeseries:
+    container:
+      type: container
+      space: '{{power_ops_models}}'
+      externalId: DayAheadProductionObligationForBenchmarking
+    containerPropertyIdentifier: timeseries
+    name: timeseries
+    description: The time series of the day ahead production obligation for benchmarking
+  name:
+    container:
+      type: container
+      space: '{{power_ops_models}}'
+      externalId: DayAheadProductionObligationForBenchmarking
+    containerPropertyIdentifier: name
+    name: name
+    description: The name of the day ahead production obligation for benchmarking
```

Comment:
**issue:** You're implementing the BidSource, which is already using a container, and then overriding the exact same properties to use a different container. I don't think this is what you intended to do.

There's also a conflict between the definitions of those two containers so you have this error.
![Screenshot 2024-06-07 at 15 28 56](https://github.com/cognitedata/power-ops-sdk/assets/22182903/ea860225-50b5-42bd-a911-4198ff610468)

Reply:
Good catch.